backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -1,35 +1,65 @@
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="oadp-backing-up-applications-restic-doc"]
-= Backing up applications with Restic
+= Backing up applications with File System Backup: Kopia or Restic
 include::_attributes/common-attributes.adoc[]
 :context: backing-up-applications
 
 toc::[]
 
-If your cloud provider does not support snapshots or if your applications are on NFS data volumes, you can create backups by using Restic.
+OADP enables you to back up and restore Kubernetes volumes attached to pods from the file system of the volumes, called File System Backup (FSB) or Pod Volume Backup. The data movement is accomplished by using modules from open source backup tools, Restic and Kopia.
+
+If your cloud provider does not support snapshots, or if your applications are on NFS data volumes, you can create backups by using FSB.
 
 [NOTE]
 ====
-link:https://restic.net/[Restic] is installed by the OADP Operator by default.
+link:https://restic.net/[Restic] is installed by the OADP Operator by default. If you prefer, you can install link:https://kopia.io/[Kopia] instead.
 ====
 
-Restic integration with OADP provides a solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to OADP’s capabilities, not a replacement for existing functionality.
+FSB integration with OADP provides a solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to OADP’s capabilities, not a replacement for existing functionality.
 
-You back up Kubernetes resources, internal images, and persistent volumes with Restic by editing the `Backup` custom resource (CR).
+You back up Kubernetes resources, internal images, and persistent volumes with Kopia or Restic by editing the `Backup` custom resource (CR).
 
 You do not need to specify a snapshot location in the `DataProtectionApplication` CR.
 
+[NOTE]
+====
+In OADP version 1.3 and later, you can use either Kopia or Restic for backing up applications.
+
+For the Built-in DataMover you must use Kopia.
+
+In OADP version 1.2 and earlier, you can only use Restic for backing up applications.
+====
+
 [IMPORTANT]
 ====
-Restic does not support backing up `hostPath` volumes. For more information, see link:https://{velero-domain}/docs/v{velero-version}/restic/#limitations[additional Restic limitations].
+FSB does not support backing up `hostPath` volumes. For more information, see link:https://velero.io/docs/v1.12/file-system-backup/#limitations[FSB limitations].
 ====
 
 .Prerequisites
```

Comment:
The only real prerequisite in this section is `You must install the OpenShift API for Data Protection (OADP) Operator.` 

Everything else should be part of the procedure, including the example code (which btw should not be part of a prerequisites section). 

One way to incorporate these items would be to add a first step to the procedure with 3 sub-steps, like this:

```
. Ensure that your <thing> is ready for <procedure> by taking the following actions:
** <one>
** <two>
** <three>
+
.Example
```

You'll need to work on the exact wording based on the functionality you're describing.

Reply:
Except for the Example, this is the standard formula used in this documentation for Prerequisites. This is how it appears in many places in the documentation. I'm going to remove the Example.